openlibrary/templates/account/books.html

```diff
@@ -61,7 +61,9 @@
       $ component_times['Yearly Goal Banner'] = time()
       $ year = get_reading_goals_year()
       $ current_goal = get_reading_goals(year=year)
-      $if not current_goal:
+
+      $# Is date between 1 Dec and 1 Feb?
+      $if not current_goal and within_date_range(12, 1, 2, 1):
         <div class="page-banner page-banner-body page-banner-mybooks">
           Announcing Yearly Reading Goals: <a href="https://blog.openlibrary.org/2022/12/31/reach-your-2023-reading-goals-with-open-library" class="btn primary">Learn More</a> or <a class="btn primary set-reading-goal-link" data-ol-link-track="MyBooksLandingPage|SetReadingGoal"href="javascript:;">$:_('Set %(year)s reading goal', year=year)</a>
```

Comment:
In future years, I wonder if this should just say, "Yearly Reading Goals" instead of "Announcing Yearly Reading Goals".

Reply:
This is a great note!